scripts/core/pattern_detector.py

```
@@ -562,58 +341,62 @@ async def detect_patterns(
+
+
+async def _classify_with_fallback(
+    members: list[Learning],
+    classifier: PatternClassifier | None,
+) -> str:
+    """Use classifier callback if provided, fall back to heuristic."""
+    if classifier is not None:
+        try:
+            return await classifier(members)
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)